src/test/java/core/utils/WaiterUtils.java

```diff
@@ -0,0 +1,30 @@
+package core.utils;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class WaiterUtils {
+
+    public static void waitElementToBeVisible(WebDriver driver, WebElement element) {
+        new WebDriverWait(driver, 30).until(ExpectedConditions.visibilityOf(element));
+    }
+
+    public static void waitForPageLoad(WebDriver driver) {
+        Wait<WebDriver> wait = new WebDriverWait(driver, 30);
+        wait.until((ExpectedCondition<Boolean>) wdriver -> ((JavascriptExecutor) driver).executeScript(
+                "return document.readyState"
+        ).equals("complete"));
+    }
+
+    public static void waitOneOfTwoElementsPresence(WebDriver driver, WebElement element1, WebElement element2) {
```

Comment:
delete if it does not work and you don't use it

Reply:
deleted